docs/getting_started.md

```diff
@@ -203,6 +203,136 @@ See these resources for more information on clustering:
+
+```r
+# submitter-provided annotations for each cell
+processed_sce$submitter_celltype_annotation
+```
+
+To access submitter-provided annotations in the `AnnData` object, use the following command:
+
+```python
+# submitter-provided annotations for each cell
+processed_adata.obs["submitter_celltype_annotation"]
+```
+
+Cells that submitters did not annotate are labeled with `Submitter-excluded`.
+Note that submitter-provided annotations are also present in unfiltered and filtered objects and can be accessed using the same approach shown here for processed objects.
+
+
+#### `SingleR` annotations
+
+`SingleR` annotation uses a reference dataset from the [`celldex` package](https://bioconductor.org/packages/release/data/experiment/html/celldex.html) [[Aran _et al._ (2019)](https://doi.org/10.1038/s41590-018-0276-y)].
+
+
+To access automated `SingleR` annotations as cell type names and/or ontology terms in the process `SingleCellExperiment` object, use the following command(s):
+
+```r
+# SingleR annotatins as cell type names
+processed_sce$singler_celltype_annotation
+
+# Or, SingleR annotatins as cell type ontology terms
+processed_sce$singler_celltype_ontology
+```
+
+To access automated `SingleR` annotations as cell type names and/or ontology terms in the processed `AnnData` object, use the following command(s):
+
+```r
+# SingleR annotatins as cell type names
+processed_adata.obs["singler_celltype_annotation"]
+
+# Or, SingleR annotatins as cell type ontology terms
+processed_adata.obs["singler_celltype_ontology"]
+```
+
+Cells that `SingleR` could not confidently annotate are labeled with `NA`.
+
+You can also access the full object returned by `SingleR` from the `SingleCellExperiment`'s metadata with the following command (note that this information is not provided in the `AnnData` object):
+
+```r
+# SingleR full result
+metadata(processed_sce)$singler_results
+```
+
+Additional information about `SingleR` annotation, including the reference source and version, is also available from the processed `SingleCellExperiment` object's metadata and from the processed `AnnData` object's `uns` slot, as described in the {ref}`experiment metadata table<sce_file_contents:SingleCellExperiment experiment metadata>`.
+
+
+#### `CellAssign` annotations
+
+`CellAssign` annotation uses a reference set of marker genes from the [`PanglaoDB` database](https://panglaodb.se/) [[Oscar Franzén _et al._ (2019)](https://doi.org/10.1093/database/baz046)], as compiled by the Data Lab for a given tissue group.
+
+To access automated `CellAssign` annotations in the `SingleCellExperiment`, use the following command:
+
+```r
+# CellAssign annotations for each cell
+processed_sce$cellassign_celltype_annotation
+```
+
+Additional information about `CellAssign` annotation results are also available from the `SingleCellExperiment`'s metadata, as described in the {ref}`experiment metadata table<sce_file_contents:SingleCellExperiment experiment metadata>`.
+
```

Comment:
```suggestion
```
I don't think you need this twice.

Reply:
I wonder if we should put 1 sentence about this, to cover both singler and cellassign, above the subsections where we go through each method. Something like, "in addition to what is described below, more information is available blah blah blah". What do you think?

Reply:
I like that! I don't think you need that and this sentence that's at the end. I would just do one, but maybe repetition isn't bad when we are talking about user facing docs.

Reply:
> I don't think you need that and this sentence that's at the end.

I agree, I think just 1 at the top is ok! In this case, the repetition will probably make it all too long.